asteroid/models/dcunet.py

```diff
@@ -17,7 +17,7 @@ class BaseDCUNet(BaseEncoderMaskerDecoder):  # CHECK-JIT
 
     """
 
-    masknet_class = DCUMaskNet
+    masknet_class = NotImplemented
```

Comment:
What happened here?

Reply:
It was bad in the initial implementation; the subclasses override this attribute and it doesn't make sense to have it in the parent class IMO